plugin/testdata/externalPluginSchema.json

```diff
@@ -1134,7 +1134,7 @@
         "children": [
             "aws.ec2InstanceConsoleOutput",
             "aws.ec2InstanceMetadataJSON",
-            "volume.FS"
+            "aws.volume.FS"
```

Comment:
This change seems strange. Also not sure what caused it. It's not the plugin prefix, that'd be `aws::`.

Reply:
Oh, that was to make testing easier. I do a global find/replace in https://github.com/ekinanp/wash/blob/namespace-type-ids/plugin/externalPluginEntry_test.go#L919 and that was originally failing b/c the `volume` entries were not prefixed with `aws.`. I figured it was easiest to just add that prefix, but I guess I could also add another `Replace` line there (or maybe use a regex instead).

I like the global find/replace there b/c it makes the test symmetric with the operation it's testing (e.g. the schema captured in the schema graph is the user's schema but w/ methods replaced by actions and type IDs namespaced by plugin-name).

Reply:
Ah. Might be trickier to regenerate.

Reply:
I guess regenerating the fixtures wouldn't hurt, given the previous type ID changes of making the paths unique.

Reply:
Actually I think I'll do that later, when it is needed.